src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -53,17 +55,59 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
     });
   }, [history, fetchMore, id, error, networkStatus]);
```

Comment:
data should be added as a dependency here

Reply:
removed data as a dependency altogether